lib/account.rb

```diff
@@ -0,0 +1,78 @@
+require 'csv'
+require 'pry'
+
+module Bank
+
+  class Account
+    attr_reader :balance, :id, :open_date
+
+
+    def initialize(id, balance, open_date)
+
+      if balance >= 0
+        @balance = balance
+      else
+        raise ArgumentError.new "insufficient funds to open account"
+      end
+
+      @id = id
+      @open_date = open_date
+    end
+
+    def withdraw(withdrawl_amount)
+
+      raise ArgumentError.new("withdrawl must be greater than 0") if withdrawl_amount < 0
+
+      if @balance - withdrawl_amount >= 0
+        @balance -= withdrawl_amount
+      else
+        print "your balance will be negative"
+        return @balance
```

Comment:
Generally if you're going to use `return` use is consistently everywhere you are returning a value. It's bad style to mix them, especially in the same method.